va/va.go

```diff
@@ -534,6 +556,80 @@ func (va VAImpl) fetchHTTP(identifier string, token string) ([]byte, string, *ac
 	return body, url.String(), nil
 }
 
+func (va VAImpl) getTXTEntry(name string) ([]string, error) {
```

Comment:
Can you add a brief godoc comment to `getTXTEntry` that describes what it does? Something like:

```
// getTXTEntry fetches TXT entries for the given domain name using the recursive resolver located at `va.customResolverAddr`, or the default system resolver if no custom resolver addr is specified
```

Reply:
Done.